packages/telestion-client-template/package.json

```
@@ -64,6 +64,8 @@
 		"@types/react-dom": "^17.0.0",
 		"@wuespace/telestion-client-cli": "file:../telestion-client-cli",
 		"@wuespace/telestion-client-types": "file:../telestion-client-types",
+		"prettier": "^2.2.1",
```

```suggestion
		"husky": "^4.3.7",
		"prettier": "^2.2.1",
```
Please add `husky` as a development dependency, too. :wink: